### ISSUE TITLE
"No targets found beneath ..." error on empty project

### ISSUE THREAD

**Issue by nnmm** [bazelbuild/intellij#5209]
### Description of the bug:

Importing an empty Bazel project gives errors:
```
ERROR
Error:no targets found beneath ''
Error:no targets found beneath 'bazel-bin'
Error:no targets found beneath 'bazel-genfiles'
Error:no targets found beneath 'bazel-out'
Error:no targets found beneath 'bazel-testlogs'
Error:no targets found beneath 'bazel-empty_bazel'
Error:no targets found beneath '.clwb'
```

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

In an empty directory, create a `WORKSPACE` file with content `workspace(name = "my_ws")`. 

Import this project and select the option to create a .bazelproject file from scratch. Proceed with the default .bazelproject file. 

### Which Intellij IDE are you using? Please provide the specific version.

CLion 2023.2

### What programming languages and tools are you using? Please provide specific versions.

-

### What Bazel plugin version are you using?

2023.07.21.0.1-api-version-232

### Have you found anything relevant by searching the web?

No.

**Comment by lbcjbb** [comment]
Same error in IntelliJ UE.

The plugin executes this command below:
```
Command: bazel query \
  --tool_tag=ijwb:IDEA:ultimate \
  --output=label_kind \
  --keep_going \
  "attr(\"tags\", \"^((?!manual).)*$\", //...:all - //bazel-bin/...:all - //bazel-genfiles/...:all - //bazel-out/...:all - //bazel-testlogs/...:all - //bazel-<projectname>/...:all - //.ijwb/...:all)" \
  --
Loading: 0 packages loaded
ERROR: Skipping '//bazel-bin/...:all': no targets found beneath 'bazel-bin'
ERROR: Skipping '//bazel-genfiles/...:all': no targets found beneath 'bazel-genfiles'
ERROR: Skipping '//bazel-out/...:all': no targets found beneath 'bazel-out'
ERROR: Skipping '//bazel-testlogs/...:all': no targets found beneath 'bazel-testlogs'
ERROR: Skipping '//bazel-<projectname>/...:all': no targets found beneath 'bazel-go'
```

But all these excluded directories in IDEA are ignored and unrecognised by Bazel.

**Comment by dhalperi** [comment]
Same bug in IntelliJ (community, ultimate). These directories are all in `.bazelignore` and properly ignored, yet appear as problems in the `bazel problems` tab in IntelliJ.

**Comment by dhalperi** [comment]
I wonder if it's the space after the `-`.

**Comment by SpencerC** [comment]
Here's the call that causes the issue:
https://github.com/bazelbuild/intellij/blob/bbebf459566f9b28214461c90efaeb3125b5f375/base/src/com/google/idea/blaze/base/sync/SyncProjectTargetsHelper.java#L124-L128

This function is called at the top of the file if [`derive_targets_from_directories`](https://ij.bazel.build/docs/project-views.html#derive_targets_from_directories) is set to `true`. Indeed, disabling this setting eliminates the error.

**Comment by LeFrosch** [comment]
Should be fixed by #6377
